esphome/components/animation/__init__.py

```diff
@@ -117,7 +117,7 @@ async def to_code(config):
                 data[pos] = rgb & 255
                 pos += 1
 
-    elif config[CONF_TYPE] == "BINARY":
+    else:  # config[CONF_TYPE] == "BINARY"/ "TRANSPANT_BINARY":
```

Comment:
I would rather keep as an elif for any future changes
```suggestion
    elif config[CONF_TYPE] in ["BINARY", "TRANSPARENT_BINARY"]:
```

Reply:
Okay, how about using a switch construction?

Reply:
Switch would be fine too